dotnet/aspnetcore-runtime-2.1/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                aspnetcore-runtime-2.1
+version             2.1.27
+revision            0
+categories          dotnet
+license             MIT
+maintainers         {tsabirgaliev @tsabirgaliev} openmaintainer
+
+description         ASP.NET Core is a cross-platform .NET framework for building modern cloud-based web applications on Windows, Mac, or Linux.
+
+long_description    .NET is a free, cross-platform, open source developer platform \
+                    for building many different types of applications. \
+                    \
+                    With .NET, you can use multiple languages, editors, and libraries \
+                    to build for web, mobile, desktop, games, and IoT.
+
+homepage            https://dotnet.microsoft.com/
+platforms           darwin
+supported_archs     x86_64
+distname            aspnetcore-runtime-${version}-osx-x64
+master_sites        https://dotnetcli.azureedge.net/dotnet/aspnetcore/Runtime/${version}/
+
+checksums           rmd160  9c801205795efa5529c2aea823f4f96b83872c5c \
+                    sha256  18c2ea1f5da7a977e46000617b8d80b14ade569e728a5d0468237b124a5d94c8 \
+                    size    58547763
+
+worksrcdir          ${name}-${version}
+extract.mkdir       yes
+
+use_configure       no
+
+depends_run         port:dotnet-cli
+
+build {}
+
+destroot {
+    set dotnet_home ${prefix}/share/dotnet
+    set runtime_dir /shared/Microsoft.AspNetCore.App
+    xinstall -d ${destroot}${dotnet_home}${runtime_dir}
+
+    copy ${worksrcpath}${runtime_dir}/${version} ${destroot}${dotnet_home}${runtime_dir}
```

Comment:
I guess `move` instead of `copy` would be better here ?

Reply:
why? I don't see many uses of `move worksrcpath -> destroot` in other Portfiles

Reply:
It just makes sense no ? If ${worksrcpath} is going to be wiped after installation, why perform the additional I/O of a copy compared to a move. Just because other ports use copy doesn't mean it wouldn't make sense there either in most cases.

For sure there are plenty that do use move ...

```
Oberon ~/Projects/MacPorts/ports > git grep move | grep destroot | grep work
<snip>
science/geant4/Portfile:                move ${workpath}/${data.name}${data.version} ${destroot}${geant.datadir}
science/libad9361-iio/Portfile:    move ${destroot}${prefix}/lib/ad9361.framework ${destroot}${frameworks_dir}
science/mpi-doc/Portfile:    move ${worksrcpath}/man/man1 ${destroot}${prefix}/share/man/
science/mpi-doc/Portfile:    move ${worksrcpath}/man/man3 ${destroot}${prefix}/share/man/
<snip>
```